ocs_ci/ocs/acm/acm.py

```diff
@@ -0,0 +1,153 @@
+
+        """
+        self.navigate_clusters_page()
+        self.do_click(self.page_nav["Import_cluster"])
+        self.do_send_keys(
+            self.page_nav["Import_cluster_enter_name"], text=f"{cluster_name}"
+        )
+        self.do_click(self.page_nav["Import_mode"])
+        self.do_click(self.page_nav["choose_kubeconfig"])
+        log.info(f"Coping Kubeconfig {kubeconfig_location}")
+        kubeconfig_to_import = copy_kubeconfig(kubeconfig_location)
+        log.info(kubeconfig_to_import)
+        self.do_click(self.page_nav["Kubeconfig_text"])
+        self.do_send_keys(
+            self.page_nav["Kubeconfig_text"], text=f"{kubeconfig_to_import}"
+        )
+        log.info(f"Submitting import of {cluster_name}")
+        self.do_click(self.page_nav["Submit_import"])
+
+
+def copy_kubeconfig(file):
+
+    try:
+        with open(file, "r") as f:
+            txt = f.readlines()
+            return txt
+
+    except FileNotFoundError:
+        log.error("file not found")
+
+
+def get_acm_url():
+    """
+    Gets ACM console url
+
+    Returns:
+        str: url of ACM console
+
+    """
+    mch_cmd = OCP(namespace="open-cluster-management")
+    url = mch_cmd.exec_oc_cmd(
+        "get route -ojsonpath='{.items[].spec.host}'", out_yaml_format=False
+    )
+    log.info(f"ACM console URL: {url}")
+
+    return f"https://{url}"
+
+
+def validate_page_title(driver, title):
+    """
+    Validates Page HTML Title
+    Args:
+        driver: driver (Selenium WebDriver)
+        title (str): required title
+
+    """
+    WebDriverWait(driver, 60).until(ec.title_is(title))
+    log.info(f"page title: {title}")
+
+
+def login_to_acm():
+    """
+    Login to ACM console and validate by its title
+
+    Returns:
+        driver (Selenium WebDriver)
+
+    """
+    url = get_acm_url()
+    log.info(f"URL: {url}, {type(url)}")
+    driver = login_ui(url)
+    acm_title = "Red Hat Advanced Cluster Management for Kubernetes"
+    validate_page_title(driver, title=acm_title)
+
+    return driver
+
+
+def verify_running_acm():
+    """
+    Detect ACM and its version on Cluster
+
+    """
+    mch_cmd = OCP(namespace="open-cluster-management")
+    acm_status = mch_cmd.exec_oc_cmd(
+        "get mch -o jsonpath='{.items[].status.phase}'", out_yaml_format=False
+    )
+    assert acm_status == "Running", f"ACM status is {acm_status}"
+    acm_version = mch_cmd.exec_oc_cmd(
+        "get mch -o jsonpath='{.items[].status.currentVersion}'", out_yaml_format=False
+    )
+    log.info(f"ACM Version Detected: {acm_version}")
+
+
+def import_clusters_with_acm():
+    """
+    Run Procedure of: detecting acm, login to ACM console, import 2 clusters
+
+    """
+    verify_running_acm()
+    driver = login_to_acm()
+    acm_nav = AcmAddClusters(driver)
+    acm_nav.import_cluster(
+        cluster_name=CLUSTER_NAME_1,
+        kubeconfig_location=KUBECONFIG_A,
+    )
+    time.sleep(300)  # Will be replace with dynamic checker.
```

Comment:
Following UI wait can also be used as suggested by Aman: https://github.com/red-hat-storage/ocs-ci/blob/fc7bb0309227e32aadcb042dc6de9973358ff3c5/ocs_ci/ocs/ui/base_ui.py#L274